samcli/lib/remote_invoke/remote_invoke_executor_factory.py

```diff
@@ -127,6 +129,41 @@ def _create_lambda_boto_executor(
             log_consumer=log_consumer,
         )
 
+    def _create_stepfunctions_boto_executor(
+        self,
+        cfn_resource_summary: CloudFormationResourceSummary,
+        remote_invoke_output_format: RemoteInvokeOutputFormat,
+        response_consumer: RemoteInvokeConsumer[RemoteInvokeResponse],
+        log_consumer: RemoteInvokeConsumer[RemoteInvokeLogOutput],
+    ):
+        """Creates a remote invoke executor for Step Functions resource type based on
+        the boto action being called.
+
+        :param cfn_resource_summary: Information about the Lambda resource
+        :param remote_invoke_output_format: Response output format
+        :param response_consumer: Consumer for writing responses to stdout
+        :param log_consumer: Log consumer for writing logs to stderr
+
+        :return: Returns the created remote invoke Executor
+        """
+        LOG.info(f"Invoking Step Function {cfn_resource_summary.logical_resource_id}")
```

Comment:
nit and feel free to ignore: personally I prefer using `%s` over f-string for logging. 
There's a pylint rule [W1203](https://pylint.readthedocs.io/en/latest/user_guide/messages/warning/logging-fstring-interpolation.html) that catches using f-string in logging, even though I think we didn't enable it.
Discussion about why `%s` is preferred for logging over f-string: https://stackoverflow.com/questions/54367975/python-3-7-logging-f-strings-vs

Reply:
I agree using `%s` rather than f-strings. Most of the logging frameworks have improvements using their formatting since they can decide whether the string needs to be formatted or not. If you use f-strings, the string will be formatted no matter the log will be printed or not (depending on the logging level).